build_ios.sh

```diff
@@ -16,9 +16,84 @@
 #
 # Builds and runs the tests, meant to be used on a bash environment.
 
+usage(){
+    echo "Usage: $0 [options]
+ options:
+   -b, build path              default: ios_unity
+   -s, source path             default: .
+   -p, framework platform      default: ${SUPPORTED_PLATFORMS[@]}
+   -a, framework architecture  default: ${SUPPORTED_ARCHITECTURES[@]}
+ example: 
+   build_scripts/ios/build.sh -b ios_build -s . -a arm64"
+}
+
+set -e
+
+readonly SUPPORTED_PLATFORMS=(device) # simulator) only support device arch for now
+readonly SUPPORTED_ARCHITECTURES="arm64;armv7" #;x86_64;i386" only support device arch for now
+#readonly DEVICE_ARCHITECTURES="arm64;armv7" only support device arch for now
+#readonly SIMULATOR_ARCHITECTURES="arm64;x86_64;i386" only support device arch for now
+
+# build default value
+buildpath="ios_unity"
+sourcepath="."
+platforms=("${SUPPORTED_PLATFORMS[@]}")
+
 # Enable utf8 output
 export LANG=en_US.UTF-8
 
+# check options
+IFS=',' # split options on ',' characters
+while getopts ":b:s:a" opt; do
```

Comment:
I believe the colon needs to be after the letter, as it is a way of saying the previous letter has options that go with it.  Also, missing p and h.  Also, can you add a c like I did for the android one, that just gets passed to the cmake configure step directly?

Reply:
Done